compiler/rustc_codegen_cranelift/src/intrinsics/mod.rs

```diff
@@ -1261,13 +1244,10 @@ fn codegen_regular_intrinsic_call<'tcx>(
             );
         }
 
-        _ => {
-            fx.tcx
-                .dcx()
-                .span_fatal(source_info.span, format!("unsupported intrinsic {}", intrinsic));
-        }
```

Comment:
What will the error message be for intrinsics that are neither implemented by cg_clif nor have a fallback body?